openspec/schemas/spec-driven-tdd/templates/spec.md

```
@@ -0,0 +1,20 @@
+## ADDED Requirements
+
+### Requirement: <!-- requirement name (SHALL/MUST) -->
+<!-- requirement text -->
+
+#### Scenario: <!-- cenário positivo -->
+- **GIVEN** <!-- estado inicial -->
+- **WHEN** <!-- ação -->
+- **THEN** <!-- resultado esperado -->
+
+#### Scenario: <!-- cenário negativo -->
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)